composables/content-render.ts

```diff
@@ -6,6 +6,7 @@ import { RouterLink } from 'vue-router'
 import { decode } from 'tiny-decode'
 import type { ContentParseOptions } from './content-parse'
 import { parseMastodonHTML } from './content-parse'
+import Emoji from '~/components/emoji/Emoji.vue'
```

Comment:
Maybe a more explicit name, like `EmojiTitleWrapper` or something?

Reply:
That feels wrong to me 😕 
Either is it a real wrapper, because it renders directly the component, nor is it potentially hard-related to the title
I tried to align the name with what already is inside Elk's components folder

So I could imagine that someone could add more features to emojis in the future, e.g. not using the `title` attribute, but translate it directly to plain text for e.g. a11y reasons and this is togglable via option/flag 🤷 

Also please note that the `title` can also be undefined if nothing matching was found (it's potentially a rar case), but so it is not a `TitleWrapper`

If you just want to satisfy the vue lint rule [multi-word-component-names](https://eslint.vuejs.org/rules/multi-word-component-names.html), why is it not active in the first place and also why do other components also don't follow that 🤔
Like e.g. `list/Account` or `Dropdown`

So what I could suggest and only comes into my mind, would be `EmojiContent`, but TBH I'm not sure if it is worth these 7 additional letters
What counts more for me is the feature itself
When I go to MS-Teams, GitHub, or nearly every other tool, I have this feature, and so I miss it in Elk

Reply:
@antfu how should we go forward?